src/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+#   Copyright 2021 Comcast Cable Communications Management, LLC
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set(PROJ_CPEABS cpeabs)
+
+if (BUILD_YOCTO)
+add_definitions(-DBUILD_YOCTO)
+endif (BUILD_YOCTO)
+
+if (PLATFORM STREQUAL "DEVICE_GATEWAY")
+add_definitions(-DDEVICE_GATEWAY)
+elseif (PLATFORM STREQUAL "DEVICE_EXTENDER")
+add_definitions(-DDEVICE_EXTENDER)
+endif ()
+
+set(HEADERS ../include/cpeabs.h)
```

Comment:
Instead of relative path use CMAKE_SOURCE_DIR  or PROJECT_SOURCE_DIR

Reply:
done